<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentXConfig</class>
 <widget class="QDialog" name="AgentXConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 38, 38);
font: 10pt;
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>501</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Source Code Pro&quot;;
background-color: rgb(143, 190, 212);
color: #2e5669;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="Tabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Source Code Pro&quot;;
color: #8fbed4;
background-color: rgb(31, 38, 38);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 38, 38);</string>
    </property>
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QFrame" name="GeneralF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Source Code Pro&quot;;
background-color: rgb(46, 86, 105);
color: #8fbed4;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="DCtrl_Check">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Directly Control Game</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="EKF_Check">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>EKF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>10</pointsize>
        <weight>2</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 23
pt &quot;Source Code Pro&quot;;
</string>
      </property>
      <property name="text">
       <string>General</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="SendUDP_Check">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Send UDP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="IP_Input">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000.000.000.000;_</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="DetectDura_Input">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Detect Duration</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Port_Input">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
     <widget class="QSlider" name="SlerpRate_Input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FSA/PnP Slerp Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FSA/PnP Pitch Offset</string>
      </property>
     </widget>
     <widget class="QSlider" name="FSAPnPOffset_input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="FSAPnPOffset_disp">
      <property name="geometry">
       <rect>
        <x>243</x>
        <y>290</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Landmark Model</string>
      </property>
     </widget>
     <widget class="QSlider" name="LandmarkModel_input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FAST/NOISY</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SLOW/ACCURACY</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Source Code Pro&quot;;
background-color: rgb(46, 86, 105);
color: #8fbed4;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25pt &quot;Source Code Pro&quot;;
</string>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="CameraID_Input">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FPS</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="FPS_Input">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>341</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Source Code Pro&quot;;
background-color: rgb(46, 86, 105);
color: #8fbed4;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25pt &quot;Source Code Pro&quot;;
</string>
      </property>
      <property name="text">
       <string>Hotkeys</string>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey1_Joystick">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="text">
       <string>JoyStick1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey1_Button">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey2_Joystick">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="text">
       <string>JoyStick2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey2_Button">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Bind_HotKey1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bind</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Bind_HotKey2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bind</string>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey1_Joystick_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Joy:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey1_Joystick_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Joy:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey1_Joystick_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CenterBtn:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Hotkey1_Joystick_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PauseBtn:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>EKFConfig</string>
    </attribute>
    <widget class="QWidget" name="ekfconf" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>691</width>
       <height>451</height>
      </rect>
     </property>
     <widget class="EKFConfig" name="ekfconfig" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>841</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>About</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EKFConfig</class>
   <extends>QWidget</extends>
   <header>ekfconfig.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AgentXConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AgentXConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
